### ISSUE TITLE
Clicking variable reporters doesn't emit 'stackclick' event

### ISSUE THREAD

**Issue by paulkaplan** [LLK/scratch-blocks#1160]
### Expected Behavior

_Please describe what should happen_

Clicking on a regular reporter in the vertical playground with event logging on shows 3 events, including the `stackclick` event that is used by the VM to run a block. 
```
Blockly.Events.Ui {group: "", recordUndo: false, element: "selected",....
Blockly.Events.Ui {blockId: "#E+{@fR+6VwE.AD[Q6!1", ....
Blockly.Events.Ui {blockId: "#E+{@fR+6VwE.AD[Q6!1", ....
```

### Actual Behavior

_Describe what actually happens_

Creating a new variable and clicking the reporter in the flyout (or the workspace) only emits a single event and no `stackclick`. 

```
Blockly.Events.Ui {group: "", recordUndo: false, element: "selected", ...
```

---

This is the cause of variable reporters not showing their values when they are clicked. It was tough to track down because click-to-report-value is broken due to other issues. But the variables were broken long before that, and this seems to be the cause.

**Comment by rachel-fenichel** [comment]
Is this only variable reporters, or all blocks?  I'm only seeing one event when I click on a block in the playground, regardless of type.

Edit: two events: selected and click.  But no stackclick.  
One event if I had already selected that block and not clicked away.

**Comment by paulkaplan** [comment]
Sorry you are right, I was on a pre-merge version of scratch blocks. I think the merge broke all stack-clicks. I'm going to revert the change that caused it, I believe it was the way that i merged https://github.com/LLK/scratch-blocks/commit/971c0c83167ba531bf23b44522a4cd5f0acfb573 which caused all stack clicks to go away.

**Comment by paulkaplan** [comment]
Once that is fixed, it still won't emit stackclicks for variable reporters because the field is marked as "EDITABLE", which I think is the root cause for this problem (editable fields are handled differently in  `Gesture#handleUp`)

**Comment by paulkaplan** [comment]
This PR reverted the event merging commit which caused all stack clicks not to work: https://github.com/LLK/scratch-blocks/pull/1161

Still need to handle the editable field one too.

**Comment by paulkaplan** [comment]
Closed by #1163
